test/unit/function/bessel_kr.cpp

```diff
@@ -0,0 +1,85 @@
+//======================================================================================================================
+/*
+  Kyosu - Complex Without Complexes
+  Copyright : KYOSU Contributors & Maintainers
+  SPDX-License-Identifier: BSL-1.0
+*/
+//======================================================================================================================
+#include <kyosu/kyosu.hpp>
+#include <test.hpp>
+
+TTS_CASE_WITH ( "Check kyosu::bessel_kr over real positive order"
+              , kyosu::scalar_real_types
+              , tts::generate(tts::randoms(-10,10))
+              )
+  <typename T>(T )
```

Comment:
If you don't use the value, just use `TTS_CASE_TPL`

Reply:
done